unified_planning/engines/factory.py

```diff
@@ -800,6 +838,37 @@ def Replanner(
             problem=problem,
         )
 
+    def PortfolioSelector(
+        self,
+        *,
+        name: Optional[str] = None,
+        params: Optional[Dict[str, Any]] = None,
+        problem_kind: ProblemKind = ProblemKind(),
+        operation_mode: Optional[Union["OperationMode", str]] = None,
+    ) -> "up.engines.engine.Engine":
```

Comment:
I think that other parameters could be useful, for example `optimality_guarantee` in case the operation mode is `OperationMode.ONESHOT_PLANNER`.